client/components/Price/Price.js

```diff
@@ -0,0 +1,38 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import styles from './Price.css';
+
+export default class Price extends PureComponent {
+  static propTypes = {
+    price: PropTypes.string,
+    onChange: PropTypes.func,
+  };
+
+  handleOnChangeAction = (e) => {
```

Comment:
the logic of this function is complicated. it needs tests. 

refactoring MIGHT be needed.